README.md

```diff
@@ -63,7 +63,7 @@
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- [Live Demo](https://crypto-currencies-b0dx.onrender.com)
+- [Live Demo](https://crypto-1xp0.onrender.com)
 - [Demo Presentation](https://drive.google.com/file/d/1YxVo_DJmcVPy6TmPbWQx7ScBk69y-6A4/view?usp=share_link)
 
```

Comment:
- [ ] OPTIONAL
Kindly include appropriate instructions under the Getting Started section of your ReadMe, especially on how to _run_ and _ run test_ cases for this project.